concepts/hedging-agents/README.md

```diff
@@ -52,7 +52,7 @@ $$
 
 When the collateral price increases (with respect to the asset stablecoins are pegged to), besides their margin (amount brought initially), HAs are entitled to get the capital gains they would have made if they had owned the collateral they hedged.
 
-If the HA brought 1 wETH and decided to back 1 wETH from the protocol, at a wETH price of 2000€, then:
+If HAs brought 1 wETH and decided to back 1 wETH from the protocol, at a wETH price of 2000€, then:
```

Comment:
Here also we should keep singular: it should not be aggregated otherwise people loose the feeling of what happens if I do this: explanation needs to be at the user perspective not the protocol